api/interestr/website/templates/website/group-details.html

```diff
@@ -481,43 +477,156 @@ <h5 class="modal-title" id="createTemplateModalLabel">Create a template</h5>
       }
     }
 
+    function replaceTextareaWithSelect() {
+      if (selectEditorTextarea) {
+        selectEditorTextarea.textContent = selectEditorTextarea.textContent.trim();
```

Comment:
To be able to update the options in the dropdown I had to change the RHS to "selectEditorTextarea.value.trim();".  Again, I was testing in Firefox version 56.0.2.